core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -16,19 +16,16 @@
 import alluxio.conf.Source;
 import alluxio.exception.status.AlluxioStatusException;
 import alluxio.exception.status.UnavailableException;
-import alluxio.network.thrift.BootstrapClientTransport;
-import alluxio.network.thrift.ThriftUtils;
-import alluxio.thrift.GetConfigurationTOptions;
-import alluxio.thrift.MetaMasterClientService;
+import alluxio.grpc.ConfigProperty;
+import alluxio.grpc.GetConfigurationPOptions;
+import alluxio.grpc.MetaMasterClientServiceGrpc;
+import alluxio.grpc.Scope;
 import alluxio.util.ConfigurationUtils;
-import alluxio.wire.ConfigProperty;
-import alluxio.wire.Scope;
+import alluxio.util.grpc.GrpcChannel;
```

Comment:
this wrapper over netty channel can be removed?

Reply:
Yes. But I am thinking of moving authentication logic to these wrappers. They will become AuthenticatedGrpcChannel and AuthenticatedGrpcServer. WDYT?